docs/install_guides/installing-openvino-runtime.md

```diff
@@ -18,6 +17,5 @@ You can choose how to install OpenVINO™ Runtime according to your operating sy
 * [Install OpenVINO Runtime on Linux](installing-openvino-linux-header.md)
 * [Install OpenVINO Runtime on Windows](installing-openvino-windows-header.md)
 * [Install OpenVINO Runtime on macOS](installing-openvino-macos-header.md)
-* [Install OpenVINO Runtime on Raspbian OS](installing-openvino-raspbian.md)
 
```

Comment:
Same as above.

Reply:
Done in https://github.com/openvinotoolkit/openvino/pull/12190/commits/187fc1f52ad6c79e76fa7bb5d48387dc8b6879d6